tests/runtest.sh

```diff
@@ -1025,6 +1059,9 @@ do
         --no-lf-conversion)
             noLFConversion=ON
             ;;
+        --build-overlay-only)
+            buildOverlayOnly=ON
```

Comment:
OK, I added a check condition just after finishing the command-line option processing.